modules/admin-top-bar/assets/scss/admin.scss

```diff
@@ -124,9 +124,11 @@ $top-bar-height: 50px;
 		}
 	}
 
-	&:not(.top-bar-active ) ~ #wpbody {
-		.wrap h1 {
-			display: block;
+	&.e-admin-top-bar--not-active ~ #wpbody {
```

Comment:
```suggestion
	&.e-admin-top-bar--inactive ~ #wpbody {
```

OR

```suggestion
	&.e-admin-top-bar--disabled ~ #wpbody {
```

Or maybe even remove that "state modifier", depends on the use case.

Reply:
We need this modifier, renamed "inactive".